### DISCUSSION TITLE
Two Projects, One Repo

### DISCUSSION THREAD

**Discussion by coallusthegreat** [vercel/community#1801] [Help] [answered]
### Summary

I have a git repository that contains two projects. Vue.JS (/client) and NestJS(/api). Both are subdirectories. I use Concurrently to run both servers under the same script locally. 

I have two vercel projects, project-frontend and project-backend, linked to the same repo but the root directory is different. I got everything working perfectly. Frontend can send requests to backend and so forth. I am using a api.domain.com subdomain for the backend. 

I'm wondering if I can merge both Vercel projects into one. When importing the repo into vercel it doesn't let me configure them differently. I tried making a vercel.json and specifying two build paths and the appropriate routes but the vue.js server got a "500 server crashed" and the logs tell me all the .vue files are just one big syntax error. The NestJS server worked though.

If you have any ideas on how I can achieve this please let me know. ChatGPT has me going in circles right now. You're my last hope.

**Comment by leerob** [comment, marked as answer]
These will need to remain as two separate Vercel projects, since they have different settings for how to route requests to each individual piece (frontend and backend). We are thinking about ways to improve this experience further in the future, but for now, two projects should work for you.
